cli/run.go

```diff
@@ -54,11 +56,13 @@ func RunRun(r *cmd.RootCMD, c *cmd.CMD) {
 	if gFlags.Debug {
 		log.SetLevel(level.Debug)
 	}
+
 	log.Debugln("Started usysconf")
+
 	defer log.Debugln("Exiting usysconf")
 	// Root user check
 	if os.Geteuid() != 0 {
-		log.Fatalln("You must have root privileges to run triggers")
+		log.Panicf("You must have root privileges to run triggers")
```

Comment:
I'm not sure panicking is the right approach. Panic should be for programming errors, not users'.

Reply:
I generally agree, however, an `os.Exit(1)` as done by `Fatalln` is not possible in this case as there are `defer` calls to execute. A panic allows these calls to execute. To ensure it is not actually a user's problem all panic's are handled in `main()` by recovering an exiting with an error. Dirty, but it captures the original intent of this code without the bug caused by `Fatalln`.

The best flow would be to return an error and show that error in main. That does require some significant overhauling, which I consider outside of the scope of this PR, especially since we are already working towards that.

Reply:
Understood. Well played then!